constants/domain_groups.json

```diff
@@ -1,841 +1,846 @@
 {
-    "Academic Papers": [
```

Comment:
Looks like you changed the spacing which makes it hard to read what the edits are?

Reply:
Same for a couple of the files. You can see when the diff is thousands of lines: https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/pull/98/files